File: 1020.cs
using System;

class MainClass {
  public static void Main (string[] args) {
    int tempo = Convert.ToInt32(Console.ReadLine());
    int year = 0;
    int mouth = 0;
    int day = 0;

    while (tempo > 0)
    {
      if(tempo >= 365)
      {
         year++;
         tempo -= 365;
      }

      else if(tempo >= 30)
      {
        mouth++;
        tempo -= 30;
      }

      else if(tempo >= 1)
      {
        day++;
        tempo -= 1;
      }
    }

    Console.WriteLine(year + " ano(s)");
    Console.WriteLine(mouth + " mes(es)");
    Console.WriteLine(day + " dia(s)");
  }
}

File: 1020.cpp
#include <iostream>

using namespace std;

int main() {
  int N;
  int day = 0;
  int days = 0;
  int month = 0;
  int year = 0;
  cin >> N;
  
  for(int i = 0; i < N; i++)
  {
      day++;
      days++;
      if(day == 30)
      {
         month++;
         day = 0;
         if(month == 12)
            month = 0;
      }
      if(days == 365)
      {
      year++;
      days = 0;
      day = 0;
      }
  }
  cout << year << " ano(s)\n";
  cout << month << " mes(es)\n";
  cout << day << " dia(s)\n";
}